jbpm-designer-api/pom.xml

```diff
@@ -43,6 +43,26 @@
 
   <build>
     <plugins>
+      <plugin>
```

Comment:
@mbiarnes Hi. You may need to add ```jbpm-designer``` to your i18n CI script to run this goal after fetching from zanata... @tsurdilo You may want to email Michael B about this; otherwise the copy may not run (there's a BASH script Michael has on the CI servers used for i18n synchronisation with Zanata; it runs the copy/rename plugin for kie-wb-distributions).